### PR TITLE
Improvements in Ansible and Jenkins jobs

### PR DESCRIPTION
**Pull request by olavph**

### CODE REVIEW

File: jenkins_jobs/trigger_nightly_host_os_build/post_build_script.sh [hunk 1]
```diff
@@ -1,16 +1,17 @@
 NIGHTLY_DIR_NAME=$(cat NIGHTLY_DIR_NAME)
 BUILD_TIMESTAMP=$(cat BUILD_TIMESTAMP)
 BUILD_DIR_PATH="../builds/${BUILD_TIMESTAMP}"
+NIGHLTY_DIR_RSYNC_URL="${UPLOAD_SERVER_USER_NAME}@${UPLOAD_SERVER_HOST_NAME}:${UPLOAD_SERVER_NIGHTLY_DIR}"
+
+rsync_upload() {
+    rsync -e "ssh -i ${HOME}/.ssh/upload_server_id_rsa" \
+          --verbose --compress --links --stats --times --chmod=a+rwx,g+wx,o- \
+          $@ $ISO_DIR_RSYNC_URL
```
Comment (farosas) [inline jenkins_jobs/trigger_nightly_host_os_build/post_build_script.sh]: Isn't `$ISO_DIR_RSYNC_URL` out of place here?

**Review by farosas — CHANGES_REQUESTED** [review]

File: jenkins_jobs/trigger_nightly_host_os_build/post_build_script.sh (hunk 1 above)
Comment (olavph) [reply jenkins_jobs/trigger_nightly_host_os_build/post_build_script.sh]: Yep. Fixing.

File: jenkins_jobs/trigger_nightly_host_os_build/post_build_script.sh [hunk 2]
```diff
@@ -1,16 +1,17 @@
...
 
-rsync -e "ssh -i $HOME/.ssh/${UPLOAD_SERVER_USER_NAME}_id_rsa" \
-      --verbose --compress --links --times --chmod=a+rwx,g+rwx,o- \
-      "$NIGHTLY_DIR_NAME" \
-      "${UPLOAD_SERVER_USER_NAME}@${UPLOAD_SERVER_HOST_NAME}:${UPLOAD_SERVER_NIGHTLY_DIR}/"
-rsync -e "ssh -i $HOME/.ssh/${UPLOAD_SERVER_USER_NAME}_id_rsa" \
-      --verbose --compress --links --times --chmod=a+rwx,g+rwx,o- \
-      "latest" \
-      "${UPLOAD_SERVER_USER_NAME}@${UPLOAD_SERVER_HOST_NAME}:${UPLOAD_SERVER_NIGHTLY_DIR}/"
+rsync_upload "$NIGHTLY_DIR_NAME" "$NIGHLTY_DIR_RSYNC_URL/"
```
Comment (farosas) [inline jenkins_jobs/trigger_nightly_host_os_build/post_build_script.sh]: Maybe the idea is to have only `$@` up there?

**Review by farosas — CHANGES_REQUESTED** [review]

File: jenkins_jobs/trigger_nightly_host_os_build/post_build_script.sh (hunk 2 above)
Comment (olavph) [reply jenkins_jobs/trigger_nightly_host_os_build/post_build_script.sh]: The idea was actually to remove the second parameter, like in `upload_build_artifacts` and `upload_iso`. Fixing.

**Review by farosas — APPROVED** [review]

**Comment by mopsfelder** [comment]
@olavph Nice improvements.  Thanks!

**Pull request merged**